### DISCUSSION TITLE
Empty viewfinder

### DISCUSSION THREAD

**Discussion by ReinBijlsma** [icosa-gallery/open-brush#178] [Q&A] [answered]
I'm using the latest Open Brush port from inside Unity. If I press the play button in Unity, I can use Open Brush with my custom made brushes, but the viewfinder window of my camera stays empty, so it's a wild guess of what I'm recording.  What do I have to edit to change this? I'm not skilled in Unity to find out myself

**Comment by andybak** [comment]
I can't reproduce this problem.

Hard to know what to suggest. Can you post *exactly* the steps you are following?

i.e.

1. I go to https://github.com/icosa-gallery/open-brush-docs/tree/master
2. I click the green "Code" button and download a zip
3. I unzip the contents and open it using Unity 2019.x.x
4. I hit the play button in Unity
5. My headset is a x y z
6. I click the camera icon on the Tools panel and swipe until it says "Video"
7. Instead of the expected preview of the camera view I see a black rectangle.

**Comment by mikeskydev** [comment, marked as answer]
Is this due to postEffects being set in your config file? #45
